test/manual/amp-sidebar-amp-list-test.html

```diff
@@ -0,0 +1,506 @@
+<!doctype html>
```

Comment:
delete this new file. add this amp-list (may be more than 1 amp-list) to the bottom of the existing amp-sidebar example